File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1114.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

senha = parseInt(lines.shift())
while(senha!=2002){
    console.log("Senha Invalida")
    senha = parseInt(lines.shift())
}
console.log("Acesso Permitido")

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1114.cs
using System;

class URI
{
    static void Main() {
        
        int senha = int.Parse(System.Console.ReadLine().Trim());
        while(senha!=2002){
            Console.Write("Senha Invalida\n");
            senha = int.Parse(System.Console.ReadLine().Trim());
        }
        Console.Write("Acesso Permitido\n");
    
    }
}